shared/services/ec-manager.go

```diff
@@ -166,6 +166,16 @@ func (p *ExecutionClientManager) PendingNonceAt(ctx context.Context, account com
 	return result.(uint64), err
 }
 
+func (p *ExecutionClientManager) GetPeerCount(ctx context.Context) (int, error) {
```

Comment:
Do you still need this, or is it a vestige of the old `rocketpool-go` PR?

Reply:
Yeah, I just removed it!